civis/imports/socrata_helpers.py

```diff
@@ -125,30 +179,51 @@ def _store_and_attach_metadata(
 
 
 def _read_paginated(
-    client,
-    dataset_id: int,
-    page_limit: int = 50000,
-    size_limit: int = None,
-    where: str = None,
+    client, dataset_id: int, page_limit: int = 90000, size_limit: int = None
 ):
+    """
+    Uses socrata API get call to pull i.
+    Pulls in Socrata data in a while loop using pandas dataframes (in 80000 row
+    chunks) that get written to .csv(s). .csv(s) and then appened together and
+    a path to final csv is outputted.
+    Inputs
+    _____________
+    dataset_id: str
+        Socrata dataset identifier
+    page_limit: int
+        number of records that can be pulled in chunk
+    date_column: 'str'
```

Comment:
These documented arguments (especially `date_column`) don't seem to match the actual arguments.